tasks/eth/014-fjord-gas-config/VALIDATION.md

```diff
@@ -0,0 +1,103 @@
+
+To allow simulating the transaction bundle of setting the required versions in a single Tenderly tx, the threshold is overridden to 1.
+Additionally, the nonce is set to 6 to account for tasks `012` and `013`.
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000004` <br/>
+  **Value:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+  **Meaning:** The threshold is set to 1. The key can be validated by the location of the `threshold` variable in the [Safe's Storage Layout](https://github.com/safe-global/safe-smart-account/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L14).
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
+  **Value:** `0x0000000000000000000000000000000000000000000000000000000000000006`
+  **Meaning:** Sets the Safe nonce to the hardcoded value of 6. This is the expected value of the Safe nonce at the time of execution. The key can be validated by the location of the `nonce` variable in the [Safe's Storage Layout](https://github.com/safe-global/safe-smart-account/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L17).
+
+## State Changes
+
+### `0x229047fed2591dbec1eF1118d64F7aF3dB9EB290` (`SystemConfig`)
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000066` <br/>
+  **Before:** `0x010000000000000000000000000000000000000000000000000c5fc500000558` <br/>
+  **After:** `0x010000000000000000000000000000000000000000000000000f79c50000146b` <br/>
+  **Meaning:** This updates the encoded `scalar` value to hold the new base fee and blob base fee scalars. See next section "Gas Config Update Details" for details.
+      The correct slot can be verified with the [storage layout](https://github.com/ethereum-optimism/optimism/blob/31653e5e51c22a239dad1a682b931e696e1539c9/packages/contracts-bedrock/snapshots/storageLayout/SystemConfig.json#L48) (`0x66 = 102`).
+
+More background on `ProtocolVersions` state validation can be found
+[here](../../common/protocol-versions.md), including a description of expected event emissions.
+
+The `SystemConfig` also emits a `ConfigUpdate` event, with `updateType` set to `1` and data containing the packed `overhead` and `scalar` values.
+The `updateType = 1` value can be verified in the contract source's [enum definition](https://github.com/ethereum-optimism/optimism/blob/31653e5e51c22a239dad1a682b931e696e1539c9/packages/contracts-bedrock/src/L1/SystemConfig.sol#L26).
+
+### Nonce increments
+
+The only other state changes are two nonce increments:
+
+- One on the Foundation Upgrade Safe (`0x847B5c174615B1B7fDF770882256e2D3E95b9D92`). If this is not decoded, it corresponds to key `0x05` on a `GnosisSafeProxy`.
+- One on the owner on the account that sent the transaction.
+
+## Gas Config Update Details
+
+The basefee and blob basefee scalars are updated to
+* `L1BaseFeeScalar: 5227`
+* `BlobBaseFeeScalar: 1014213`
```

Comment:
What is the most 'official' source we can point to for how these numbers were chosen? 

I was expecting to find them in the governance proposal.

Reply:
If you look at the [similar Ecotone proposal](https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/eth/005-2-ecotone-set-gas-config), we also referenced the calculation sheet as the primary source, together with the input parameters. These scalar values are actually not voted on, and don't require a gov proposal/vote to be changed.

Reply:
> we can ask @MSilb7 for additional confirmation.

[Source of scalars](https://docs.google.com/spreadsheets/d/1V3CWpeUzXv5Iopw8lBSS8tWoSzyR4PDDwV9cu2kKOrs/edit?gid=186414307#gid=186414307), which should be publicly linked in docs as well.

Inputs are based on 28 days of OP Mainnet data, targeting a 5% margin on L1 Data and FP State Proposal Costs (Note: does not yet account for improved brotli compression).

Reply:
If possible, we should re-evaluate scalars 1 week / 1 month post-Fjord. To observe any behavioral differences & compression improvements with brotli.